Assets/Shopify/Examples/Scripts/ShopPopup.cs

```diff
@@ -31,6 +31,7 @@ public class ShopPopup : MonoBehaviour {
 
     public string AccessToken = "8f40b2ede3e02be97a81ac29cfabc6e0";
     public string ShopDomain = "unity-buy-sdk.myshopify.com";
+    public string Locale = null;
```

Comment:
Shouldn't this be `en`? Also, initializing a string object as `null` is redundant (at least in C#)

Reply:
Removed the assignment. 👍